### ISSUE TITLE
Sourcemapping is offset during debug

### ISSUE THREAD

**Issue by brion-fuller** [monounity/karma-coverage-istanbul-instrumenter#5]
Sourcemapping doesn't seem to update to account for instrumentation.  I did look into the issue.  I did notice that when calling instrument the sourcemap looks exactly the same as the original.  Some how this isn't being updated which seems weird as karma-typescript uses a similar call which works.

I have a sample repo available to highlight the issue and better debugging.  I will look a little more into this issue but was hoping you would have some better knowledge on this.

First test I saw this in was using "cobertura" reports.  You can see that most functions are labelled as (anonymous_0). These should show somevar, sum, and substract.

Second part is when I am trying to debug a test in chrome debugger.  Steps to see the case.

1. clone repo
2. run npm install
3. run npm test
4. Click debug button
5. Open debug tools
6. Place a breakpoint in substract or add
7. You will noticed in the callstack that you are not in the right location and variables are not defined correctly.

**Comment by erikbarke** [comment]
Do you have a link to the sample repo?

**Comment by brion-fuller** [comment]
Yeah sorry I forgot that. https://github.com/brion-fuller/karma-coverage-istanbul-instrumenter. 
I did find one work around but seemed weird.  So I would get the source map from the instrument.lastSourceMap.  I noticed this was different from the original.  So when appended to the output it worked for me.
